build.sbt

```diff
@@ -16,6 +16,21 @@ def scalaReleasesForBinaryVersion(v: String): Seq[String] = v match {
     )
 }
 
+def scalaStdlibForBinaryVersion(v: String): Seq[String] = {
+  val commonLibs = Seq(
+    "nativelib",
+    "clib",
+    "posixlib",
+    "windowslib",
+    "javalib",
+    "auxlib"
+  )
+  v match {
+    case "2.12" | "2.13" => commonLibs :+ "scalalib"
+    case "3"             => commonLibs :+ "scala3lib"
```

Comment:
In the case of the Scala 3 it would not be correct. For Scala 3 we need both `scalalib_2.13` and `scala3lib_3`. 
It is because Scala 3 reuses Scala 2.13 standard library (eg. package `scala.collection`) and extends it with some new classes defined only in Scala 3 standard library, eg. package `scala.deriving`